Jenkinsfile

```diff
@@ -697,9 +697,10 @@ pipeline {
                         }
                     }
                 }
-                stage('win-RS1') {
+                stage('win-RS1-master') {
```

Comment:
don't think we have to rename the stage

Do we know if Jenkins has a param that says "it's a PR"? I think we want to run these on release branches as well, so basically what we want to filter on is; "don't run this on PR jobs"

Reply:
Looks like there is; https://stackoverflow.com/a/48103602/1811501

(and then remove `branch 'master'` from the example below)

```groovy
        when {
            allOf {
                environment name: 'CHANGE_ID', value: ''
                branch 'master'
            }
        }
```

Reply:
And the `allOf` looks not needed; https://jenkins.io/doc/book/pipeline/syntax/

> If the `when` directive contains more than one condition, all the child conditions must return true for the stage to execute. This is the same as if the child conditions were nested in an `allOf` condition

Reply:
oh; this is more elegant; and the official way; https://jenkins.io/doc/book/pipeline/syntax/

```groovy
when {
    not { changeRequest() }
}
```

Reply:
i've updated